From: Stephen Sinclair <radarsat1@gmail.com>
Date: Wed, 28 Dec 2016 15:33:18 -0300
Subject: Remove install instructions for renderer.

---
 externals/CMakeLists.txt | 27 ---------------------------
 1 file changed, 27 deletions(-)

diff --git a/externals/CMakeLists.txt b/externals/CMakeLists.txt
index 270adf9..f00ed0c 100644
--- a/externals/CMakeLists.txt
+++ b/externals/CMakeLists.txt
@@ -87,33 +87,6 @@ library_project_setup()
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/FCMangle.h DESTINATION include/${PROJECT_NAME})
 
 
-if(HAVE_SICONOS_MECHANICS)
-  # This files is not installed as a program, even though it is one.
-  # Remove the comment if you know how it should be installed --xhub
-  #  configure_file(io/SimpleGui.py ${SICONOS_SWIG_ROOT_DIR}/io/SimpleGui.py @ONLY) 
-
-
-  file(GLOB rendererFiles RELATIVE ${CMAKE_SOURCE_DIR}/externals/renderer  ${CMAKE_SOURCE_DIR}/externals/renderer/img/*.*)
-  foreach(rendererFile ${rendererFiles})
-    set(srcRendererPath ${CMAKE_SOURCE_DIR}/externals/renderer/${rendererFile})
-    if(NOT IS_DIRECTORY ${srcRendererPath})     
-      install(FILES  ${srcRendererPath} DESTINATION share/siconos/renderer/img)
-    endif(NOT IS_DIRECTORY ${srcRendererPath})
-  endforeach(rendererFile)
-
-  file(GLOB rendererFiles RELATIVE ${CMAKE_SOURCE_DIR}/externals/renderer/  ${CMAKE_SOURCE_DIR}/externals/renderer/threeJS_libraries/*.*)
-  foreach(rendererFile ${rendererFiles})
-    set(srcRendererPath ${CMAKE_SOURCE_DIR}/externals/renderer/${rendererFile})
-    if(NOT IS_DIRECTORY ${srcRendererPath})     
-      install(FILES  ${srcRendererPath} DESTINATION share/siconos/renderer/threeJS_libraries  )
-    endif(NOT IS_DIRECTORY ${srcRendererPath})
-  endforeach(rendererFile)
-
-  install(PROGRAMS renderer/renderer.py
-    DESTINATION bin RENAME siconos_renderer)
-  
-endif()
-
 if(WITH_SERIALIZATION)
   install(FILES
     ${CMAKE_CURRENT_SOURCE_DIR}/boost_contribs/boost/serialization/unordered_collections_load_imp.hpp
